lib/components/src/typography/DocumentFormatting.tsx

```diff
@@ -25,24 +25,29 @@ const withMargin: CSSObject = {
 export const H1 = styled.h1<{}>(withReset, headerCommon, ({ theme }) => ({
   fontSize: `${theme.typography.size.l1}px`,
   fontWeight: theme.typography.weight.black,
+  color: theme.color.defaultText,
```

Comment:
This can be put in headerCommon

Reply:
thanks, great feedback. Updated `headerCommon` and pushed